src/compiler/diagnosticMessages.json

```diff
@@ -1951,6 +1951,10 @@
         "category": "Error",
         "code": 2690
     },
+    "Cannot find name '{0}'. A type exists with this name, but no value.": {

```

Comment:
Another idea is to get rid of the "cannot find name" portion and try something more direct out:

`'{0}' only refers to a type, but is being used as a value here.`

@RyanCavanaugh and @danquirk you have a lot of UX opinions, what do you think?